src/chrome/content/rules/Bundesfinanzhof.de.xml

```diff
@@ -0,0 +1,17 @@
+<!-- Federal Fiscal Court -->
+<ruleset name="Bundesfinanzhof.de">
+
+	<target host="bundesfinanzhof.de" />
+	<target host="www.bundesfinanzhof.de" />
+	<target host="juris.bundesfinanzhof.de" />
+
+	<securecookie host=".+" name=".+" />
+
+	<!-- Invalid certificate on https://bundesfinanzhof.de/,
```

Comment:
Works for me.

Reply:
The cert is only valid for `www.bundesfinanzhof.de`. On Firefox I get `SSL_ERROR_BAD_CERT_DOMAIN`.